### PR TITLE
renamed specs files to add them in rake test

### PR DESCRIPTION
**Pull request by VenkateshRavula**
### Description
All the spec file names should have suffix "_spec" to the filenames, which is missed for few files and those files are ignored during UTs execution. Changed the file names as required.

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
  - [ ] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [ ] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

**Comment by soodpr** [comment]
Please add below test case also under this PR:
https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/enclosure/import_certificate.rb

**Comment by chebroluharika** [comment]
@soodpr I have added UTs for enclosure.

File: spec/unit/resources/enclosure/import_certificate_spec.rb [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
     {
       'type' => 'CertificateDataV2',
-      'base64Data' => 'encrypted_data'
+      'base64Data' => nil
     }
   end
 
   it 'imports a signed certificate' do
+    allow(File).to receive(:open).and_call_original
+    allow(File).to receive(:open).with('csr_data_file.txt').and_return(true)
```
Comment (soodpr) [inline spec/unit/resources/enclosure/import_certificate_spec.rb:17]: Not sure if we are using this file content in importing the certificate below. Can you please check to read and parse the file contents

**Review by soodpr — APPROVED** [review]
LGTM except enclosure import certificate spec which can be fix later.

**Review by nabhajit-ray — APPROVED** [review]

**Pull request merged**